app/models/facility.rb

```diff
@@ -243,15 +218,12 @@ def teleconsultation_enabled?
   end
 
   def teleconsultation_phone_number_with_isd
-    teleconsultation_phone_number = teleconsultation_phone_numbers.first
-    return if teleconsultation_phone_number.blank?
-
-    Phonelib.parse(teleconsultation_phone_number.isd_code + teleconsultation_phone_number.phone_number).full_e164
+    teleconsultation_phone_numbers_with_isd.first&.dig(:phone_number)
```

Comment:
Grab the first MO's number instead of the facility attribute. This method powers the `api/v4/facility_teleconsultations` endpoint that may still be in use.

Reply:
Should we nudge praveen once again to send out a notification to update apps? This change might surprise users on the oldest teleconsult version.

Reply:
As discussed, I'll clarify the impacts with Praveen before shipping this cleanup.

Reply:
Clarified. No comms necessary, we can ship this.